tests/todo/test_migration.py

```
@@ -0,0 +1,306 @@
+    new_id = r1["lists"][0]["new_id"]
+    todo_list = await todo_store.get_list(new_id)
+    assert todo_list is not None
+    assert len(todo_list["items"]) == 3
+
+
+@pytest.mark.asyncio
+async def test_migrate_recovers_from_interruption(shared_store, todo_store):
+    """If migration was interrupted after target creation but before source
+    deletion, a re-run detects the existing matching list, flows entries into
+    it, and stamps ``migrated_from`` so future re-runs skip by exact id."""
+    doc, entries = await _setup_list_doc(
```

Ruff (0.15.21)

[warning] Unpacked variable `entries` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)